docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,31 +6,40 @@ Please include a summary of the change and which issue is fixed. Please also inc
 
 Please delete options that are not relevant.
 
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Documentation update
-- [ ] Code cleanliness or refactor
-- [ ] Dependency change
+- Bug fix (non-breaking change which fixes an issue)
+- New feature (non-breaking change which adds functionality)
+- Documentation update
+- Code cleanliness or refactor
+- Dependency change
 
 ## Screenshot
 
-Please provide if applicable.
+Please provide if applicable (browser for `frontend` or any visual way to show a `backend` change).
 
 # Testing
 
-Manual steps to test this functionality:
+Applicable for all code changes.
+
+Manual steps to reproduce this functionality:
+
+1.  Go to \_\_\_ page.
+2.  Click \_\_\_. Notice \_\_\_.
 
 Any concerns that need to be addressed in product usability testing:
 
+- Concern.
+
 Screenshot of unit tests run passing:
 
-# Checklist:
+# Checklist
+
+Applicable for all code changes.
 
 - [ ] My code follows the style guidelines of this project
 - [ ] I have commented my code, particularly in hard-to-understand areas
 - [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new compiler warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
-- [ ] I have researched WCAG2.1 standards and met them in this PR
-- [ ] My changes look good on IE
+- [ ] I have researched WCAG2.1 accessibility standards and met them in this PR
```

Comment:
Recommending that we add "(want to add your proposed change "(can be navigated using a keyboard)" as you proposed in #71 (rather than having 2x PRs modifying the same file). Or we can discuss when you come in